com.unity.netcode.gameobjects/Runtime/Core/NetworkManager.cs

```diff
@@ -225,7 +225,7 @@ internal void SetSessionOwner(ulong sessionOwner)
                 foreach (var networkObjectEntry in SpawnManager.SpawnedObjects)
                 {
                     var networkObject = networkObjectEntry.Value;
-                    if (networkObject.IsOwnershipSessionOwner && LocalClient.IsSessionOwner)
+                    if (networkObject.IsOwnershipSessionOwner && networkObject.OwnerClientId != LocalClientId)
```

Comment:
This needs further review as it was originally a client-side adjustment made for session owner but needs to include a new version of the service in order to know that it should skip handling this on the client side.

Reply:
My bad... looks good.